website/docs/r/transfer_server.html.markdown

```diff
@@ -61,12 +61,29 @@ resource "aws_iam_role_policy" "example" {
 }
 POLICY
 }
+
+resource "aws_transfer_server" "foo" {
+  identity_provider_type = "SERVICE_MANAGED"
+  logging_role           = "${aws_iam_role.foo.arn}"
+  protocols              = ["SFTP"]
+
+  tags = {
+    NAME = "tf-acc-test-transfer-server"
+    ENV  = "test"
+  }
+}
 ```
 
 ## Argument Reference
 
 The following arguments are supported:
 
+* `certificate` - (Optional) The Amazon Resource Name (ARN) of the AWS Certificate Manager (ACM) certificate. This is required when `protocols` is set to `FTPS`
+* `protocols` - (Optional) Specifies the file transfer protocol or protocols over which your file transfer protocol client can connect to your server's endpoint. This defaults to `SFTP`
+The available protocols are:
```

Comment:
This doesn't render in the documentation as a bulleted list; Maybe just list the valid values as a comma-separated list.